view/frontend/web/template/payment/offsite-installment-form.html

```diff
@@ -64,7 +64,12 @@
                             </div>
                         </label>
                         <select style="margin-top:10px" data-bind="options: getInstallmentTerms('ktc'),
```

Comment:
Would this be better in CSS? This hardcoded styling may break someone's theme?

Reply:
this is some past change outside of this PR. Will add it to todo list for future pr